README.md

```diff
@@ -250,31 +250,30 @@ xcode-select --install
 ```
 Follow the prompts to continue.
 
-To install GCC and CMake on macOS we recommend to use [Homebrew](https://brew.sh/) package manager. With Homebrew
+To install GCC, CMake and python on macOS we recommend to use [Homebrew](https://brew.sh/) package manager. With Homebrew
 installed, run the following commands:
 ```sh
-brew install gcc@14 cmake
+brew install gcc@14 cmake python3
 ```
 After installation, check `cmake --version`. If command is not found, then you need to add to environment path to
 homebrew installed packages. To do this open `~/.zprofile` (for Zsh) or
 `~/.bash_profile` (for Bash) and add to the end of the file the output of `brew shellenv`.
 After that, restart the terminal and check the version of CMake again, now it should be displayed.
 
 Then you need to install Boost library built with GCC. Please avoid using Homebrew for this, as the Boost version provided by Homebrew
-is built with Clang, which has a different ABI. Instead, download the latest version of Boost from the [official website](https://www.boost.org/users/download/) and unpack the archive to the `/usr/local/` directory or another directory of your choice:
+is built with Clang, which has a different ABI. Instead, download the latest version of Boost from the [official website](https://www.boost.org/users/download/), open terminal and run:
 ```sh
-cd /usr/local/
+cd ~/Downloads
 curl https://archives.boost.io/release/1.86.0/source/boost_1_86_0.tar.bz2 --output "boost_1_86_0.tar.bz2"
-tar xvjf boost_1_86_0.tar.bz2
-rm boost_1_86_0.tar.bz2
+tar xvjf boost_1_86_0.tar.bz2 && rm boost_1_86_0.tar.bz2
 cd boost_1_86_0
 ```
 Navigate to the unpacked Boost directory in the terminal and run the following commands:
 ```sh
 ./bootstrap.sh 
 echo "using darwin : : g++-14 ;" > user-config.jam
 ./b2 install --user-config=user-config.jam --layout=versioned
-export BOOST_ROOT=$(pwd) # export Boost_ROOT=$(pwd) for CMake 3.26 and below.
+export BOOST_ROOT=/usr/local/ # export Boost_ROOT=/usr/local/ for CMake 3.26 and below.
```

Comment:
I didn't know it before, but Boost `./b2 install` installs boost libs to usr/local/ by default. It can be override by `--prefix`.